lib/ast-converter.js

```diff
@@ -115,22 +115,47 @@ TOKEN_TO_TEXT[SyntaxKind.CaretEqualsToken] = "^=";
 TOKEN_TO_TEXT[SyntaxKind.AtToken] = "@";
 TOKEN_TO_TEXT[SyntaxKind.InKeyword] = "in";
 
+/**
+ * Returns true if the given node is a valid ESTree class member
+ * @param  {object}  node AST node
+ * @returns {boolean}      is valid class member
+ */
 function isESTreeClassMember(node) {
     return node.kind !== SyntaxKind.PropertyDeclaration && node.kind !== SyntaxKind.SemicolonClassElement;
 }
 
+/**
+ * Returns true if the given token is a comma
+ * @param  {object}  token the syntax token
+ * @returns {boolean}       is comma
+ */
 function isComma(token) {
     return token.kind === SyntaxKind.CommaToken;
 }
 
+/**
+ * Returns true if the given operator is the assignment operator
+ * @param  {object}  operator the operator
```

Comment:
Please use `{Object}` (uppercase O) for objects, but try to be more specific as per previous comments.